include/mega/command.h

```diff
@@ -28,6 +28,11 @@
 #include "http.h"
 
 namespace mega {
+
+struct JSON;
+struct MegaApp;
+class MegaStringMap;
```

Comment:
MegaStringMap is part of the public interface, and so can't be used in the core code as some clients are built without the public api (ie. megaapi_impl.cpp is not included)  It's a one-way dependency (and I made that mistake initially too :-) )

Reply:
Ok, will resolve